subsys/bluetooth/mesh/proxy.c

```diff
@@ -1157,6 +1157,7 @@ static struct bt_mesh_subnet *next_sub(void)
 		sub = &bt_mesh.sub[(i + next_idx) % ARRAY_SIZE(bt_mesh.sub)];
 		if (advertise_subnet(sub)) {
 			next_idx = (next_idx + 1) % ARRAY_SIZE(bt_mesh.sub);
+            sub->node_id = BT_MESH_NODE_IDENTITY_RUNNING;
```

Comment:
I'm not sure this is quite right. This function just checks whether subnets are ready for sending, and should not enable or disable node id advertisements. Node ID should only ever be enabled with the config server, and run for `CONFIG_BT_MESH_NODE_ID_TIMEOUT` seconds. Originally, we only enable identity for a subnet in the `node_id_start` function, which also sets the timestamp. I think the other change you made above is attempting to fix this behavior, but it looks like it's just fixing the symptom.

Reply:
You were right, it's only fixing the symptom
The problem was different - advertise was not started for the node.
Now it can start and stop if proxy disable, and after stopping it will start again.

Thanks again, you have a great OS.

I plan to add a driver for ST7735s display, I use it in one project and want to switch to this OS and LVGL
But this is another story :)